docs/source/examples/layerwise-folding.md

```diff
@@ -0,0 +1,305 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# ZNE with Qiskit: Layerwise folding
+
+
+This tutorial shows an example of how to mitigate noise on IBMQ backends with
+the Cirq frontend using layerwise folding in contrast with global folding.
+
+
+- [ZNE with Qiskit: Layerwise folding](#zne-with-qiskit-layerwise-folding)
+  - [Setup](#setup)
+  - [Helper functions](#helper-functions)
+  - [Define circuit to analyze](#define-circuit-to-analyze)
+  - [Total variational distance metric](#total-variational-distance-metric)
+  - [Impact of single vs. multiple folding](#impact-of-single-vs-multiple-folding)
+  - [Executor](#executor)
+  - [Global folding with linear extrapolation](#global-folding-with-linear-extrapolation)
+  - [Layerwise folding with linear extrapolation](#layerwise-folding-with-linear-extrapolation)
+
++++
+
+## Setup
+
+```{code-cell} ipython3
+from typing import Dict, List, Optional
+import numpy as np
+import os
+import cirq
+import qiskit
+import matplotlib.pyplot as plt
+
+from mitiq import zne
+from mitiq.zne.scaling.layer_scaling import layer_folding, get_layer_folding
+from mitiq.interface.mitiq_qiskit.qiskit_utils import initialized_depolarizing_noise
+from mitiq.interface.mitiq_qiskit.conversions import to_qiskit
+from mitiq.interface.mitiq_cirq.cirq_utils import sample_bitstrings
+
+from cirq.contrib.svg import SVGCircuit
+from qiskit_ibm_provider import IBMProvider
+
+USE_REAL_HARDWARE = False
+
+if USE_REAL_HARDWARE:
+    # Save account credentials.
+    #IBMProvider.save_account(token="<IBM_TOKEN_HERE>")
+
+    # Load a previously saved account.
+    provider = IBMProvider()
+    backend = provider.get_backend("ibmq_qasm_simulator")
+    noise_model = None
+else:
+    # Default to a simulator.
+    backend = qiskit.Aer.get_backend("qasm_simulator")
+    noise_model = initialized_depolarizing_noise(noise_level=0.02)
+
+shots = 10_000
+```
+
+**Note:** When `USE_REAL_HARDWARE` is set to `False`, a classically simulated noisy backend is used instead of a real quantum computer.
```

Comment:
Not sure if we need to support the `USE_REAL_HARDWARE` option in this example. We can remove this part and just keep:
```
backend = qiskit.Aer.get_backend("qasm_simulator")
noise_model = initialized_depolarizing_noise(noise_level=0.02)
```

Reply:
Yeah, having that extra option is nice, but it kind of obscures the main theme of the intent of this tutorial. Removed!